Add function and test for LIC number 5

apeinot opened the pull request:
Add function LIC5() and the corresponding test cases
The launch input condition 5 is described in the corresponding issue #4 
This should resolve #4

apeinot69 committed:
Add function and test for LIC number 5

Add function LIC5() and the corresponding test cases
This should resolve #4

Jonagoo commented:
Looks good to me.

francislaus commented:
Your implementation resolves #4 and provides good test cases for it. But I think you should stick to the coding standards according to our document (point 3: the opening bracket should always be in the same line as the statement it corresponds to)

apeinot69 committed:
Fix in the format of the code according to our coding standards

We have decide that the opening bracket should always be in the same line as the statement it corresponds to.
It's now fix in LIC5 function. This should resolve #4

apeinot69 committed:
Merge branch 'master' into issue4

Resolve the conflicts. Now the branch is ready to merge and close #4

apeinot commented:
@francislaus I have fix what you asked for and resolve the conflicts with master. If it's good for you, merge the pull request and close #4
